fgcm/fgcmConfig.py

```diff
@@ -469,6 +470,10 @@ def __init__(self, configDict, lutIndex, lutStd, expInfo, checkFiles=False, noOu
                                                                  float, -0.05,
                                                                  ndarray=True, required=True,
                                                                  dictName='expGrayPhotometricCutDict')
+        self.expFwhmCut = self._convertDictToBandList(self.expFwhmCutDict,
+                                                      float, 100.0,
+                                                      ndarray=True, required=False,
+                                                      dictName='expFwhmCutDict')
```

Comment:
What is 100.0 here, default cut?  If it's meant to indicate "unphysical", I might go with a 99.9 type of thing...